lib/node_modules/@stdlib/stats/base/scumin/examples/c/example.c

```diff
@@ -17,27 +17,25 @@
 */
 
 #include "stdlib/stats/base/scumin.h"
-#include <stdint.h>
 #include <stdio.h>
-#include <inttypes.h>
 
 int main( void ) {
 	// Create strided arrays:
-	float x[] = { 1.0, 2.0, -3.0, 4.0, -5.0, 6.0, 7.0, 8.0 };
-	float y[] = { 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0 };
+	const float x[] = { 1.0f, 2.0f, -3.0f, 4.0f, -5.0f, 6.0f, 7.0f, 8.0f };
+	const float y[] = { 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f };
```

Comment:
```suggestion
	float y[] = { 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f };
```

this shouldn't be constant